flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/table/utils/TiKVTypeUtils.java

```diff
@@ -201,6 +244,10 @@ public static DataType getFlinkType(final org.tikv.common.types.DataType dataTyp
                     object = LocalTime.ofNanoOfDay(Long.parseLong(object.toString()));
                 }
                 break;
+            case "OffsetDateTime":
+                object =
```

Comment:
`OffsetDateTime` should be mapped to `TIMESTAMP WITH TIMEZONE`, current implementation will loss the timezone information, we'd better use String here. BTW, `OffsetDateTime` can not be casted to `Timestamp`.